readme.md

```diff
@@ -306,6 +306,7 @@ This section details services that have integrated Lighthouse data. If you're wo
 ## Related Projects
 Other awesome open source projects that use Lighthouse.
 
+* **[Exthouse](https://github.com/treosh/exthouse)** - Analyze the impact of a browser extension on web performance.
```

Comment:
just waiting for the day someone names their project "AAA Lighthouse" to take over that prime real estate 😆

Reply:
😂

Reply:
brb making `___aaaa___a1111_Lighthouse_Analyzer`

Reply:
`<NUL>BestAppEver` wins